### ISSUE TITLE
[0.11.3/4] /usr/bin/less: unrecognized option: X

### ISSUE THREAD

**Issue by tylernathanreed** [bobthecow/psysh#717]
I'm working with a Laravel project, and I recently ran `composer update`, and I noticed that my tinker functionality had broken.

Any command (no matter what kind) appears to be producing the following error:

```
>>> $i = 0
/usr/bin/less: unrecognized option: X
BusyBox v1.34.1 (2021-11-23 00:57:35 UTC) multi-call binary.

Usage: less [-EFIMmNSRh~] [FILE]...

View FILE (or stdin) one screenful at a time

        -E      Quit once the end of a file is reached
        -F      Quit if entire file fits on first screen
        -I      Ignore case in all searches
        -M,-m   Display status line with line numbers
                and percentage through the file
        -N      Prefix line number to each line
        -S      Truncate long lines
        -R      Remove color escape codes in input
        -~      Suppress ~s displayed past EOF
RuntimeException with message 'Error closing output stream'
```

I checked the `composer.lock` file, and saw that I had upgraded from `0.11.2` to `0.11.4`.

I ran the same dummy command (`$i = 0`) on `0.11.3` and `0.11.4`, and I get the same error. This tells me that something in the `0.11.3` release broke the psy shell output for me.

I looked at the release notes, and saw that the command went from `less -R -S -F -X` to `less -R -F -X`. There's probably more to this, as the old command had the `-X` flag too, yet the shell worked fine for me.

Since this might be environment related, here's my details:

```
bash-5.1$ cat /etc/os-release
NAME="Alpine Linux"
ID=alpine
VERSION_ID=3.15.0
PRETTY_NAME="Alpine Linux v3.15"
HOME_URL="https://alpinelinux.org/"
BUG_REPORT_URL="https://bugs.alpinelinux.org/"
```

For what it's worth, I'm running Docker inside of WSL. It's possible that my Docker image is missing some core dependency or something, as my WSL environment supports the `-X` flag on the `less` command.

I'm version locking my project to `0.11.2` until this issue is resolved. Hopefully either a solution will be found here, or I can modify my environment to support whatever new feature was released.

**Comment by bobthecow** [comment]
[In `v0.11.3`](https://github.com/bobthecow/psysh/releases/tag/v0.11.3) PsySH started using output paging for stdout and return values. It used to do this, but there was a regression somewhere along the way. So I imagine you'd have your same error anywhere output paging _was_ happening in `v0.11.2` (I think `doc` and `dump` commands should do it, given long enough docs or things to dump). 

The quick fix is to explicitly define your pager, either using `cli.pager` PHP setting, or the `pager` setting in [a PsySH config file](https://github.com/bobthecow/psysh/wiki/Configuration). Since `-X` seems to be supported with most common setups, we'll probably keep it as the default.

**Comment by ceejayoz** [comment]
An `apk add --update less` in my `Dockerfile` resolved this for me on Alpine 3.15.
